docroot/profiles/custom/cgov_site/modules/custom/pdq_drug_information_summary/tests/src/Functional/PDQDrugSummaryTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Drupal\Tests\cgov_site\Functional;
+
+use Drupal\Tests\SchemaCheckTestTrait;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Tests Functionality of the PDQ Cancer Information Summary content type.
```

Comment:
... PDQ _Drug_ Information Summary ....

And since you're editing this file anyway, you should probably change the class name in the commented-out code at the bottom, as you did for the code in the CIS test file - unless it's never going to be discarded eventually, in which case just ditch it now.